pkg/mapper/examples/cat_sleep/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/numaproj/numaflow-go/pkg/mapper"
+)
+
+const DEFAULT_SLEEP_SECONDS = 10
+
+type CatSleep struct {
+}
+
+func (e *CatSleep) Map(ctx context.Context, keys []string, d mapper.Datum) mapper.Messages {
+
+	// sleep for as long as the environment variable indicates (or default if not configured)
+	sleepSeconds := DEFAULT_SLEEP_SECONDS
+	secondsString := os.Getenv("SLEEP_SECONDS")
+	if secondsString == "" {
+		log.Printf("SLEEP_SECONDS environment variable not set, using default %d seconds\n", DEFAULT_SLEEP_SECONDS)
+	} else {
+		val, err := strconv.Atoi(secondsString)
+		if err != nil {
+			log.Printf("SLEEP_SECONDS environment variable %q not an int, using default %d seconds\n", secondsString, DEFAULT_SLEEP_SECONDS)
+		} else {
+			sleepSeconds = val
+		}
+	}
```

Comment:
this can be done in main?

Reply:
True. I suppose I can make this purposely slow container more efficient. :D 

but it would still be neater though, I agree